ui/espresso/ScreenshotSample/app/build.gradle

```diff
@@ -31,6 +31,23 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    testOptions {
+        devices {
+            // run with ../gradlew  -Pandroid.sdk.channel=3 -Pandroid.experimental.androidTest.useUnifiedTestPlatform=true nexusOneApi30DebugAndroidTest
```

Comment:
You can omit `-Pandroid.experimental.androidTest.useUnifiedTestPlatform=true` as this is set to true by default.